rollup/bundleHelpers.js

```diff
@@ -34,6 +35,11 @@ export function makeBaseBundleConfig(options) {
   const markAsBrowserBuildPlugin = makeBrowserBuildPlugin(true);
   const licensePlugin = makeLicensePlugin(licenseTitle);
   const tsPlugin = makeTSPlugin('es5');
+  const rrwebBuildPlugin = makeRrwebBuildPlugin({
+    excludeCanvas: false,
```

Comment:
@billyvg should we exclude canvas right now for CDN bundles? 🤔

Reply:
Yeah since we don't support playback. Makes sense to me

Reply:
👍